applications/vanilla/settings/structure.php

```diff
@@ -40,6 +40,7 @@
     ->column('DateMarkedRead', 'datetime', null)
     ->column('AllowDiscussions', 'tinyint', '1')
     ->column('Archived', 'tinyint(1)', '0')
+    ->column('CanDelete', 'tinyint', '1')
```

Comment:
I chose `CanDelete` because it's a column for Groups and for Badges and is used in logic 50+ other places. `Deletable` sounds nicer but it's used exactly that once and looked an awful lot like `Deleted` which is another column we use.